MTAOS/v2/comcam.yaml

```diff
@@ -1,11 +1,5 @@
----
-
-# Default setting
-
-# Type of camera for wavefront estimation pipeline (WEP) to use
-# (lsstCam, lsstFamCam, comcam)
 camera: comcam
-
-# Type of instrument for optical feedback control (OFC) to use
-# (comcam, lsst, lsstfam)
 instrument: comcam
+visit_id_offset: 2021111900000
+collections: refcats,LSSTComCam/raw/all,LSSTComCam/calib
+data_path: /repo/LSSTComCam
```

Comment:
Please consider to add the reference or comment to let the user knows what values can be used for each setting. Otherwise, it might be very confusing to the other developers.